web/src/app/models/geometry/geometry.ts

```diff
@@ -33,3 +33,14 @@ export enum GeometryType {
   POLYGON = 'POLYGON',
   MULTI_POLYGON = 'MULTI_POLYGON',
 }
+
+/** A label for a given geometry type. Defaults to 'Polygon'. */
+export function geometryTypeLabel(geometryType?: GeometryType): string {
+  if (geometryType === GeometryType.POINT) {
```

Comment:
Can we use a switch statement or an immutable Map here?

Reply:
Yeah, swapped this for a switch statement